FirstSolution/Part15Task1P2and3/Transport.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace Part15Task1P2
+{
+    public class Transport
+    {
+        private static string transportType;
+        private string destination;
+        private int number;
+        private string dateOfDepature;
+        private int numberOfSeats;
+        
+        public Transport()
```

Comment:
No need to define empty default constructor

Reply:
Changed